InformationScripting/src/parsing/SimpleQueryParser.cpp

```diff
@@ -71,14 +71,33 @@ QueryNode* SimpleQueryParser::parseAny(const QString& queryString, int& index)
 		if (index < queryString.length())
 			ch = queryString[index];
 	}
-	else query = new CommandNode(commandString); // Create the command. Note that it could be an empty string.
+	else
+	{
+		// Create the command. Note that it could be an empty string.
+		auto parts = commandString.split(" ");
```

Comment:
At some point we might want to support strings and stuff in arguments, where spaces would be allowed, but I guess not for now.